tsl/test/expected/continuous_aggs_union_view-11.out

```diff
@@ -436,3 +436,195 @@ SELECT * FROM boundary_view ORDER BY time_bucket;
           40 | 400
 (4 rows)
 
+---- TEST union view with WHERE, GROUP BY and HAVING clause ----
+create table foo (a integer, b integer, c integer);
+select table_name FROM create_hypertable('foo', 'a', chunk_time_interval=> 10);
+NOTICE:  adding not-null constraint to column "a"
+ table_name 
+------------
+ foo
+(1 row)
+
+INSERT into foo values( 3 , 16 , 20);
+INSERT into foo values( 1 , 10 , 20);
+INSERT into foo values( 1 , 11 , 20);
+INSERT into foo values( 1 , 12 , 20);
+INSERT into foo values( 1 , 13 , 20);
+INSERT into foo values( 1 , 14 , 20);
+INSERT into foo values( 2 , 14 , 20);
+INSERT into foo values( 2 , 15 , 20);
+INSERT into foo values( 2 , 16 , 20);
+INSERT into foo values( 20 , 16 , 20);
+INSERT into foo values( 20 , 26 , 20);
+INSERT into foo values( 20 , 16 , 20);
+INSERT into foo values( 21 , 15 , 30);
+INSERT into foo values( 21 , 15 , 30);
+INSERT into foo values( 21 , 15 , 30);
+CREATE OR REPLACE FUNCTION integer_now_foo() returns int LANGUAGE SQL STABLE as $$ SELECT coalesce(max(a), 0) FROM foo $$;
+SELECT set_integer_now_func('foo', 'integer_now_foo');
+ set_integer_now_func 
+----------------------
+ 
+(1 row)
+
+CREATE OR REPLACE VIEW mat_m1( a, countb, sumbc, spreadcb, avgc )
+WITH (timescaledb.continuous, timescaledb.refresh_lag = 10)
```

Comment:
Do you need the refresh lag? Time in tests cannot be controlled reliably anyway.